File: src/RedeSocialAcademica/CadastroUsuario/Domain/Services/Strategy/PasswordValidation/MaximumFiveCharacters/Classes/SymbolsStrategy.cs
namespace CadastroUsuario.Domain.Services.Strategy.PasswordValidation.MaximumFiveCharacters.Classes
{
    public class SymbolsStrategy : IFiveCharacterStrategy
    {
        private string _symbols = "!@#$%¨&*()£¢¬/?°\\|\":;.,><^~`´}]º{{ª=+§-_\'";
        public bool CharacterValid(char character)
        {
            return _symbols.Contains(character);
        }
    }
}

File: src/RedeSocialAcademica/academysm-client/src/domain/services/passwordValidator/hasMaxFiveCharsOfSameTypeInASequence/symbolsStrategy.ts
import IMaximumCharacterStrategy from "./maximumCharacterStrategy";

export default class SymbolsStrategy implements IMaximumCharacterStrategy {
    private symbols: string = "!@#$%¨&*()£¢¬/?°\\|\":;.,><^~`´}]º{{ª=+§-_\'"

    characterValid(character: string): boolean {
        return this.symbols.includes(character)
    }

}